servicetalk-grpc-protoc/src/main/java/io/servicetalk/grpc/protoc/Generator.java

```diff
@@ -1639,6 +1645,43 @@ private TypeSpec newServiceInterfaceSpec(final State state, final boolean blocki
                 .addStatement("return $L", blocking ? BLOCKING_METHOD_DESCRIPTORS : ASYNC_METHOD_DESCRIPTORS)
                 .build());
 
+        // generate default service methods
```

Comment:
Consider adding a test in `ProtocolCompatibilityTest` to make sure our default behavior is the same as grpc-java default behavior. In grpc-netty tests we can always use this new option `defaultServiceMethods=true`, bcz we will likely make it new default in the next "major" version.